src/Mado/QueryBundle/Repositories/BaseRepository.php

```diff
@@ -146,6 +147,13 @@ public function setQueryOptionsFromRequest(Request $request = null)
         return $this;
     }
 
+    private function ensureFilterIsValid($filters )
+    {
+        if (!is_array($filters)) {
+            throw new InvalidFiltersException("Wrong query string exception: " . var_export($filters, true) . "\n Please check query string should be something like this http://127.0.0.1:8000/?filtering[status]=todo");
```

Comment:
"Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#23-lines

Reply:
@orociscaverai up

Reply:
@orociscaverai up